lib/aur-chroot

```diff
@@ -205,36 +211,73 @@ while read -r key _ value; do
     case $key=$value in
         Server=file://*)
             bindmounts_rw+=("${value#file://}") ;;
+        CacheDir=*)
+            # XXX: cachedir binding is a hairy subject
+            # see https://gitlab.archlinux.org/archlinux/devtools/-/blob/master/src/arch-nspawn.in?ref_type=heads#L86
+            bindmounts_rw+=("${value}") ;;
     esac
 done < <(pacman-conf --config "$pacman_conf")
 wait "$!"
 
 # create new container, required for update/build steps
-if (( create )); then
+if (( create )) && [[ ! -d $directory/root ]]; then
     # default to base-devel or multilib-devel, unless packages are
     # specified on the command-line.
     # available packages depend on the configured pacman configuration
     if (( $# )); then
         base_packages=("$@")
-
-    # XXX: use pacini to not process Include directives in pacman.conf
-    # (not supported by devtools)
     elif [[ $(pacini --section=multilib "$pacman_conf") ]] && [[ $machine == "x86_64" ]]; then
```

Comment:
Since we don't care about devtools now, we could use `pacconf` instead of `pacini`

Reply:
I implemented this by using pacman-conf (less dependencies) even though I don't know the differences between the two.

Reply:
The main difference is that pacconf has `--raw` ($repo and $arch are not replaced, standard values are not included in the output). But it might not be a requirement for `aur-chroot`